nnstreamer/tensor_filter/tensor_filter_nntrainer.cc

```diff
@@ -176,6 +176,7 @@ int NNTrainer::loadModel() {
 
   if (!g_file_get_contents(model_config, &content, &file_size, &file_error)) {
     ml_loge("Error reading model config!! - %s", file_error->message);
+    g_free(content);
```

Comment:
this is not needed.
As the [documentation](https://developer.gnome.org/glib/stable/glib-File-Utilities.html#g-file-get-contents) says, if error, content will be set to NULL.

Reply:
> this is not needed but will work with it too as `g_free()` accepts NULL.
> As the [documentation](https://developer.gnome.org/glib/stable/glib-File-Utilities.html#g-file-get-contents) says, if error, content will be set to NULL.

Good. then, I will remove this g_free.